src/search_engines/google/google.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json 
+
+class GoogleSearch:
+    def __init__(self, logger, api_key, cse_id):
+        self.__logger = logger
+        self.api_key = api_key
+        self.cse_id = cse_id
```

Comment:
Ocultar esses dados sensíveis

```suggestion
        self.__api_key = api_key
        self.__cse_id = cse_id
```